src/popup/App.vue

```diff
@@ -14,7 +14,7 @@
         class="app-inner"
         :class="{ 'styled-scrollbar': showScrollbar }"
       >
-        <Header v-if="isAuthenticated && showHeader" />
+        <Header v-if="showHeader" />
```

Comment:
There was a reason to hide the header when user is not authenticated - previously user was seeing for a while the remnants of the header on the login screen.

Reply:
This remnants were fixed in a different way by waiting for router to be ready.
`isRouterReady` variable is added into equation of `showHeader` computed variable.